nahusenayElias/Team_4
# DRD-93: Create Mautic rules for adding contacts to segments

## Related ticket / Customer approval:
[DRD-93](https://edu-team4-react24k.atlassian.net/browse/DRD-93?atlOrigin=eyJpIjoiYjVjM2IxNjE5YWMzNDM1MjlhYTM2YjE1NTI0Zjg3NjgiLCJwIjoiaiJ9)
## In this PR:
- Created rules for adding contacts to mautic segments "visited jobs page" and "filled contact form"
## Testing instructions:
- Checkout branch
- import mautic db (make sure to cd to `mautic-lando` folder first)
- a contact that visits jobs page or fills in the contact form should be automatically added to corresponding mautic segment
